README.md

```diff
@@ -0,0 +1,92 @@
+# Online Bookstore
+
+---
+
+### Introduction
+
+---
+
+The "Online bookstore" is a web application for purchasing books online.
+In this app, users can browse available books, search for desired books by multiple criteria, save them to a shopping cart, and place an order.
+This allows users to buy books without the need to leave their homes or browse through physical bookshelves, making the process comfortable and fast.
+
+---
+
+### Technologies and tools
+
+---
+
+- Java 21
+- Spring Boot, Spring Security, Spring Data JPA
+- MySQL, Liquibase
+- JWT
+- Maven, Docker
+- Lombok, Mapstruct, Swagger
+- Junit, Mockito, Testcontainers
+
+---
+
+### How to run:
+
+---
+

```

Comment:
Add - clone repo 
Do I need to install Java Development Kit (JDK) ?

Reply:
About JDK, we're just running a docker image here which is isolated, is it not?